apps/client/src/app/auth/signup/step1/page.tsx

```diff
@@ -7,30 +7,26 @@ import React, { useState } from "react";
 import Button from "~/components/Button/Button";
 
 export default function SignUpFirst() {
-  const [choiceUser, setChoiceUser] = useState({ userType: "" });
+  const [choiceUser, setChoiceUser] = useState<string>("");
```

Comment:
```suggestion
const [choiceUser, setChoiceUser] = useState<'buyer' | 'seller'| null>(null);
```
으로 하고, 필요할 때 null을 가드해 버리는 게 좀 더 명확하다는 생각이 듭니다 😃

Reply:
와우... 너무 좋습니다 감사합니다 !